netdisco/discoverables/webos_tv.py

```diff
@@ -13,7 +13,21 @@ def info_from_entry(self, entry):
 
     def get_entries(self):
         """Get all the LG WebOS TV device uPnP entries."""
-        return self.find_by_device_description({
-            "deviceType": "urn:dial-multiscreen-org:device:dial:1",
-            "friendlyName": "[LG] webOS TV"
-        })
+
+        # Lookup for webos3
+        webos_entry = self.find_by_device_description(
+            {
+                "deviceType": "urn:schemas-upnp-org:device:MediaRenderer:1",
+                "manufacturer": "LG Electronics.",
+                "modelName": "LG TV"
+            }
+        )
+
+        if not len(webos_entry):
```

Comment:
You mean leave only the new ssdp search I've added ?
Ok and I guess anyone with HA and and LG tv can confirm it.

I'll do it tonight when I get home